### DISCUSSION TITLE
Add macOS

### DISCUSSION THREAD

**Discussion by k3soulja** [N0chteil-Productions/Apple-Music-RPC#13] [Ideas]
Make a mac version pls

**Comment by 7loud** [comment]
Hey
thanks for your interest in AMRPC. We are sorry that you can't use AMRPC at the moment and we are trying to release AMRPC for macOS ASAP!

**Comment by 7loud** [comment]
Hey @k3soulja (and @beetlepimp0001),
we've finally finished the development for macOS! We'll release the update later today. 🎉
